.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: Release
+
+on:
+  pull_request:
+  push:
+    branches: ["master"]
+  release:
+    types: ["published"]
+
+jobs:
+  build:
+    if: ${{ github.event.pull_request.head.repo.full_name == github.repository || github.event_name != 'pull_request' }}
```

Comment:
I don't think this line is needed! Sorry if I misspoke.

I'm thinking it should push to `master` (tag) under these circumstances:

- A commit is merged directly to master
- A PR is merged into master

with those, I think we can remove this line and the `pull_request:` trigger, since `push: branches: ["master"]` should handle both those cases

Reply:
> I'm thinking it should push to master (tag) under these circumstances:

Oh sorry I missed that in your message above.